src/context.tsx

```diff
@@ -1,4 +1,4 @@
-import React, { createContext } from "react";
+import * as React from "react";
```

Comment:
So — they haven’t broken it since 2019, and it’ll remain working in React 19 🤣

I do however have a sliver of hope they may come up with ESM and tree-shaking, so I buy this idea with a caveat:

```suggestion
import { createContext } from "react";
```

This will work fine with the "new" JSX transform and if we ever get tree shaking this won’t import the whole of react with it.

Reply:
Yes, I agree with you, a priori there is time! But you never know 😄

I made the modification to use named modules, and maybe benefit from tree-shaking one day ✌️